### ISSUE TITLE
Option to pause\resume backups

### ISSUE THREAD

**Issue by TristisOris** [eduardolat/pgbackweb#55]
No it's no way to disable task.

**Comment by Stitch10925** [comment]
Yes there is. When you edit the backup you have the option to enable/disable it:

![image](https://github.com/user-attachments/assets/c6b284e5-bb5f-446b-b526-a9f3f295f82b)

**Comment by TristisOris** [comment]
oh, missed it. how about slider at backup list?
